src/dowel/csv_output.py

```diff
@@ -56,6 +54,36 @@ def record(self, data, prefix=''):
         else:
             raise ValueError('Unacceptable type.')
 
+    def _augment(self, data):
+        """Augment tabular data with new column(s)"""
```

Comment:
It's really not obvious from this method name or docstring that it modifies files. Could you make it more clear?

Reply:
Of course! I was trying to be consistent with the existing comment density, but I agree that this instance should be expanded upon.